src/common/actions/mocks.js

```diff
@@ -0,0 +1,15 @@
+import { DELETE_MOCK, MOCK_RESPONSE } from '../modules/proxy/constants';
+
+export const deleteMock = url => ({
+  type: DELETE_MOCK,
+  payload: url,
+});
+
+export const deleteAllMocks = mocks => dispatch => {
```

Comment:
Should we create an endpoint for deleting all mocks?

Reply:
yes for sure. This is in the leu of that ...